src/middleware/remove-prefix.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction } from 'express';
+
+const removePrefix = (prefix: string) => {
```

Comment:
Nice code organization.

Is is possible to change this to `removePrefixMiddleware` to help make it more clearer as to what it does?